gradlew

```diff
@@ -213,7 +213,7 @@ DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
 set -- \
         "-Dorg.gradle.appname=$APP_BASE_NAME" \
         -classpath "$CLASSPATH" \
-        org.gradle.wrapper.GradleWrapperMain \
+        -jar "$APP_HOME/gradle/wrapper/gradle-wrapper.jar" \
```

Comment:
How does it know the classname now?

Reply:
Oh, I forgot about that detail. Was part of my Java certification 25 years ago…

Reply:
Oh wow! I didn't know I was working with a _certified_ Java Developer! Awesome!